nautobot/ipam/filters.py

```diff
@@ -458,35 +511,38 @@ class Meta:
     def search(self, queryset, name, value):
         if not value.strip():
             return queryset
-        qs_filter = Q(dns_name__icontains=value) | Q(description__icontains=value) | Q(address__istartswith=value)
+        qs_filter = Q(dns_name__icontains=value) | Q(description__icontains=value)
```

Comment:
Is this a loss of capability here?

Reply:
Yes, it won't support `starts with` queries anymore. I discussed with with Ken and he didn't think it was an issue

Reply:
Could do that based on the network membership? It doesn't have to be done with substring matches, no?